pom.xml

```diff
@@ -145,12 +145,6 @@
             <version>2.7</version>
         </dependency>
         <dependency>
-            <groupId>se.jiderhamn</groupId>
-            <artifactId>classloader-leak-prevention</artifactId>
-            <version>1.11.0</version>
-            <scope>runtime</scope>
-        </dependency>
-        <dependency>
```

Comment:
There is no way to configure the classloader in iceland, it has to be done in an service implementation. The service implementation might also choose not to use the library.